creativecloud/scripts/scripts.js

```diff
@@ -14,6 +14,7 @@ import { setLibs } from './utils.js';
 
 // Add project-wide style path here.
 const STYLES = '/creativecloud/styles/styles.css';
+const base = `${window.location.origin}/creativecloud`;
```

Comment:
We do not need to create an absolute base url. You can use config codeRoot and it will get relative url from the current domain